### ISSUE TITLE
Add checksum validation for solr tar download

### ISSUE THREAD

**Issue by mark-dce** [curationexperts/ansible-hydra#80]
We should add a step that reads and confirms the checksum for the solr download.  Three times now over a few days, I've gotten a bad download that the fails on the unarchive step.  There's been no way to fix this without deleting the .tgz file and re-downloading it.

```
TASK: [hydra-stack/install | download solr tarball] *************************** 
changed: [default]

TASK: [hydra-stack/install | untar solr tarball] ****************************** 
failed: [default] => {"failed": true, "parsed": false}
BECOME-SUCCESS-zhmnceryjwalzxxurdvnrdwwuuwkzrkr
{"src": "/opt/install/solr-4.10.3.tgz", "group": "vagrant", "uid": 1000, "dest": "/opt/install/", "changed": true, "extract_results": {"cmd": "/bin/tar -xzf \"/opt/install/solr-4.10.3.tgz\"", "out": "", "err": "", "rc": 0}, "state": "directory", "handler": "TgzArchive", "mode": "0755", "owner": "vagrant", "gid": 1000, "check_results": {"cmd": "/bin/tar -C \"/opt/install/\" --diff -zf \"/opt/install/solr-4.10.3.tgz\"", "out": "", "unarchived": false, "err": "/bin/tar: solr-4.10.3/LUCENE_CHANGES.txt: Warning: Cannot stat: No such file or directory\n/bin/tar: solr-4.10.3/contrib/analysis-extras/lib: Warning: Cannot stat: No such file or directory\n/bin/tar: solr-4.10.3/contrib/clustering/lib: Warning: Cannot stat: No such file or directory\n
etc...
```

**Comment by acozine** [comment]
I see only an ASC signature, md5 and sha1 hashes for Solr: https://archive.apache.org/dist/lucene/solr/4.10.4/. In ansible 2.0, we'll be able to use "checksum", so let's revisit after we upgrade. Depends on #22.

**Comment by acozine** [comment]
Closed by #178.
